tasks/06_init_borg_repo.yml

```diff
@@ -0,0 +1,10 @@
+---
+# - name: Init repository
```

Comment:
As mentioned, the precise command depends on what key mode, encryption algo and Borg version one uses. Should be done outside the role.